samples/toolbarconfigurator/js/fulltoolbareditor.js

```diff
@@ -335,7 +335,7 @@ window.ToolbarConfigurator = {};
 			}
 		}
 
-		$button.setHtml( text );
+		$button.setHtml( CKEDITOR.tools.htmlSafeByReview(text, 'Safe only accessible internally or by developer.') );
```

Comment:
If we are sure that `text` contains no markup, is it possible to use a `setText(...)` or some `.textContent = ...` equivalent? Or, otherwise, if we are sure that this is text and not HTML, we can sanitize it.

Reply:
This is actually being used in samples/ so we don't need this at all. I'll remove the htmlSafeByReview

Reply:
LGTM